pkg/packagekit/wix/service.go

```diff
@@ -241,3 +230,18 @@ func (s *Service) Xml(w io.Writer) error {
 	return nil
 
 }
+
+// cleanServiceName removes characters windows doesn't like in
+// services names, and converts everything to snake case.
```

Comment:
> `converts everything to snake case`
Doesn't this conflict with the returned `defaultName`, which is the return value of `snaker.SnakeToCamel()`? seems like this returns camelCase strings

Reply:
Fixed comment.